lua/ulx/modules/sh/pcrestrict.lua

```diff
@@ -0,0 +1,45 @@
+CATEGORY_NAME = "CFC"
+
+local function addToWhitelist(callingPlayer, targetPlayers)
+    local affected_plys = {}
+
+    for _, ply in pairs(targetPlayers) do
+        local isPlyWhitelisted = isPlayerWhitelisted(ply)
```

Comment:
Isn't `isPlayerWhitelisted()` defined serverside?

In any case this feels weird to be shared because it's actually doing logic. I think the logic should be exclusively serverside.

Is this shared because the clients need to know that the ulx command exists? Is there a better way to do this?

Reply:
I believe the way I am doing it is the best way, its just a weird quirk about ulx... You need to define your commands in the shared realm.